Sources/BraintreePayPal/BTPayPalReturnURL.swift

```diff
@@ -0,0 +1,84 @@
+import Foundation
+
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+enum BTPayPalReturnURLState {
+    case unknownPath
+    case succeeded
+    case canceled
```

Comment:
small nit:
```suggestion
    case cancelled
```

Reply:
Our SDKs always use the 1 L canceled/cancelation